app/queues/dns/dns-flow.server.ts

```diff
@@ -82,8 +83,6 @@ export const addDnsRequest = async ({ username, type, name, value }: JobRecord)
 };
 
 export const updateDnsRequest = async ({ id, username, type, name, value }: JobRecord) => {
-  const fullRecordName = buildDomain(username, name);
```

Comment:
Question: Why did we have this line in the first place?

Reply:
It was probably copied from the record creation, but since the name is already formatted there and saved in that format there is no need to format (build) it again

Reply:
cc @Genne23v.  I wonder if we should document this and/or alter the name of the variable, since a name portion and a full domain are different.  It can happen in a follow-up vs here, but we should clarify so that `name` doesn't mean 2 different things.